src/server/controllers/user.controller.ts

```diff
@@ -236,6 +237,18 @@ export const completeOnboardingHandler = async ({
       input.step = temp_onboardingOrder.find((step) => user.onboardingSteps.includes(step));
     }
 
+    // Check recaptcha for Buzz step
+    if (input.step === 'Buzz') {
+      const { recaptchaToken } = input;
+      if (!recaptchaToken) throw throwAuthorizationError('recaptchaToken required');
+
+      const riskScore = await createRecaptchaAssesment({
+        token: recaptchaToken,
+        recaptchaAction: RECAPTCHA_ACTIONS.COMPLETE_ONBOARDING,
+      });
+      if (!riskScore || riskScore < 0.5) throw throwAuthorizationError('Recaptcha score too low');
+    }
```

Comment:
Let's do a more "general" error. `We are unable to log you in right now. Please try again later`.